project/Settings.scala

```diff
@@ -109,6 +111,21 @@ object Settings {
       Settings.Repositories.geosolutions,
       Settings.Repositories.osgeo,
       Settings.Repositories.locationtechReleases
+    ),
+    headerLicense := Some(HeaderLicense.ALv2(java.time.Year.now.getValue.toString, "Azavea")),
+    headerMappings := Map(
+      FileType.scala -> CommentStyle.cStyleBlockComment.copy(commentCreator = new CommentCreator() {
+        val Pattern = "(?s).*?(\\d{4}(-\\d{4})?).*".r
+        def findYear(header: String): Option[String] = header match {
+          case Pattern(years, _) => Some(years)
+          case _                 => None
+        }
+        def apply(text: String, existingText: Option[String]): String = {
+          // preserve year of old headers
+          val newText = CommentStyle.cStyleBlockComment.commentCreator.apply(text, existingText)
+          existingText.flatMap(_ => existingText.map(_.trim)).getOrElse(newText)
+        }
+      })
```

Comment:
Did moving the config here fix some issue with header generation?

Reply:
@echeipesh yes, it should be set per project and for some reason setting through `ThisBuild` doesn't work. I tried it in geotrellis-pointcloud & geotrellis-server.

Reply:
👍 TIL